### PR TITLE
add addNewComment component

### PR DESCRIPTION
**Pull request by valy4**

### CODE REVIEW

File: src/components/AddNewComment.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      <div className="AddComment_container">
+        <div className="Username_input">
+          <label>UserName</label>
+          <input name="username" defaultValue={addNewComment.username}></input>
+        </div>
+        <div className="Comment_input">
+          <textarea
+            name="body"
+            value={textValue}
+            onChange={(e) => setTextValue(e.target.value)}
```
Comment (popatre) [inline src/components/AddNewComment.jsx]: Nice controlled component here

File: src/components/AddNewComment.jsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    });
+  };
+
+  return (
+    <form onSubmit={handleCommentSubmit}>
+      <p>Comment:</p>
+      <div className="AddComment_container">
+        <div className="Username_input">
+          <label>UserName</label>
+          <input name="username" defaultValue={addNewComment.username}></input>
```
Comment (popatre) [inline src/components/AddNewComment.jsx]: This input needs to be a controlled component. At the moment, the browser is in charge of it and react doesnt know what inside it at all times.

File: src/components/AddNewComment.jsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      created_at: "just now",
+    });
+    if (addNewComment.body === "") {
+      return;
+    }
+
+    setComments([addNewComment, ...comments]);
+    Api.postComment(article_id, addNewComment).then(() => {
+     
+      setTextValue("");
```
Comment (popatre) [inline src/components/AddNewComment.jsx]: What happens if this fails?

**Review by popatre — CHANGES_REQUESTED** [review]
Some good code in here, just a few things to add/change, please:

- [ ] Error handling - we need some feedback if it doesnt successfully post the comment in the db i.e. if the internet is off. At the moment, the comment appears on the page, but it isnt actually in the db. You only realise something has gone wrong once you refresh and dont see the comment anymore. 
- [ ] Could you lock the post button so that i cant keep clicking it on a slow connection. I can keep clicking the button at the moment and accidentlly make multiple requests, kjust because the internet is slow. 
- [ ] Any form input needs to be controlled, so the username one also needs this, like the text area.

**Review by B7Bprog — APPROVED** [review]
This is a very nice work Valentina! 🎉  I like the optimistic rendering of comments. The error handling works fine too. Feel free to merge. 👍

**Pull request merged**